src/Files.App/ServicesImplementation/Settings/AppearanceSettingsService.cs

```diff
@@ -70,6 +71,13 @@ public String AppThemeFontFamily
 			set => Set(value);
 		}
 
+		/// <inheritdoc/>
+		public SystemBackdropType AppThemeSystemBackdrop
+		{
+			get => Get(MicaController.IsSupported() ? SystemBackdropType.MicaAlt : SystemBackdropType.Solid);
```

Comment:
Wouldn't it be better to keep MicaAlt so that it uses the fallback colors?

Reply:
That's something I missed, thanks for pointing it out